packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -36,8 +36,76 @@ const core = {
             strong: color.fadedOffBlack32,
         },
         inverse: {
-            subtle: color.offBlack16,
-            default: color.offBlack8,
+            subtle: color.fadedOffBlack16,
+            default: color.fadedOffBlack8,
+            strong: color.white,
+        },
+    },
+    background: {
+        instructive: {
+            subtle: color.fadedBlue8,
+            default: color.blue,
+            strong: color.activeBlue,
+        },
+        neutral: {
+            subtle: color.fadedOffBlack8,
+            default: color.fadedOffBlack72,
+            strong: color.offBlack,
+        },
+        critical: {
+            subtle: color.fadedRed8,
+            default: color.red,
+            strong: color.activeRed,
+        },
+        success: {
+            subtle: color.fadedGreen8,
+            default: color.green,
+            strong: color.activeGreen,
```

Comment:
The doc uses the hex directly (#00672C), does the doc need to be updated with `activeGreen`?

I'm treating the doc as the source of truth since it's the shared resource, but let me know if that's not the case!